lib/workers/repository/onboarding/pr/index.ts

```diff
@@ -99,7 +99,7 @@ If you need any further assistance then you can also [request help here](${confi
   prBody = prBody.replace('{{CONFIG}}\n', configDesc);
   prBody = prBody.replace(
     '{{WARNINGS}}\n',
-    getWarnings(config) + getDepWarnings(packageFiles)
+    getWarnings(config) + getDepWarnings(packageFiles, 'pr')
```

Comment:
```suggestion
    getWarnings(config) + getDepWarnings(packageFiles)
```
isn't "pr" the fallback value? if so, it can be omitted.

Reply:
I believe it's better to add the `'pr'` in here (but not in the tests)

Reply:
This should be separated into two different functions